src/main/resources/api/service_openapi.yaml

```diff
@@ -531,6 +535,11 @@ components:
           description: The ID of the Data Repo snapshot
           type: string
 
+    WorkspaceStageEnum:
```

Comment:
I'm not sold on `Stage`. I'm on the fence, could be convinced. WDYT about `Mode`?
Stage sounds like these workspaces are going to progress through these states. While that is the eventual migration plan, it's not going to be reflected in our code - migration probably happens outside of WM proper.

Reply:
My thinking with `Stage` (both here and in the doc) was that it makes it clear that there is a progression, even if moving through that really ends up happening outside WM. I didn't want to use something like `Mode` or `Type` because that feels like we're drawing a more permanent distinction between the workspaces, and I think that makes it more likely that it would creep into being more than a temporary feature toggle, which I'd like to avoid.